tests/Integration/Http/HttpClientInitializerTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Tempest\Integration\Http;
+
+use Tempest\HttpClient\GenericHttpClient;
+use Tempest\HttpClient\HttpClient;
+use Tests\Tempest\Integration\TestCase;
+
+class HttpClientInitializerTest extends TestCase
+{
+    public function test_container_can_initialize_http_client()
+    {
+        $httpClient = $this->container->get(HttpClient::class);
```

Comment:
After rebase on main, you'll have to manually call `$this->container->addInitializer` here

Reply:
Huh. It won't automatically discover it?

Reply:
Oh, it's an integration test, nvm 👍

PS: we should rename `Integration\TestCase` to `IntegrationTest`